Core/uWebshop.Domain/Model/OrderDTO/Order.cs

```diff
@@ -44,7 +44,7 @@ public class Order : IConvertibleToOrderInfo
 	    public DateTime? FulfillDate;
 		public PaymentInfo PaymentInfo;
 		public int PaymentProviderPrice;
-		public int? PaymentProviderOrderPercentage;
+		public decimal? PaymentProviderOrderPercentage;
```

Comment:
No I don't need more places, but the end user should be able to enter the value as 2.80% or 0.028 instead of 280 (which is confusing).